packages/SwingSet/test/test-controller.js

```diff
@@ -113,6 +116,10 @@ test('bootstrap', async t => {
   // basedir-controller-2/bootstrap.js logs "bootstrap called" and queues a call to
   // left[0].bootstrap
   const c = await buildVatController(config);
+  for (let i = 0; i < 5; i += 1) {
+    // eslint-disable-next-line no-await-in-loop
+    await c.step(); // vat starts
```

Comment:
I bet this can be c.run

Reply:
Nope.

Reply:
Huh, that test is really confused. I'll rewrite it after this lands. The 5 steps you've got here are enough to deliver `startVat` and `bringOutYourDead` to two other vats, and just `startVat` to a third (but not `bringOutYourDead`), leaving one vat half-finished, a second vat completely unfinished, and the bootstrap method doesn't even get delivered. Which is good, because the `bootstrap` vat it references emits `'bootstrap called'` when in fact only `buildRootObject` has been called, and it doesn't even define a `bootstrap()` method, so when `c.run()` gets all the way through, it crashes.

Reply:
I added #4632 to fix this one.